config.proto

```diff
@@ -46,6 +46,9 @@ message Reporting {
 
     // opa describes the setting related to the Open Policy Agent
     Opa opa = 4;
+
+    // reporter type - zipkin or otlp. Defaults to Zipkin, in the future it will change to otlp.
```

Comment:
How about enum instead? otherwise this becomes a whatever value thing. WE could also make it possible to support console and file.

Reply:
> How about enum instead?

I will have a look at enum

> WE could also make it possible to support console and file.

This can be done incrementally per use-case.

Reply:
There is this use case already https://github.com/hypertrace/agent-config/issues/6

Reply:
Let's tackle that in a separate PR.